### DISCUSSION TITLE
Download Failed

### DISCUSSION THREAD

**Discussion by a314941** [aclap-dev/video-downloadhelper#719] [Help and Q&A]
### Your question

I keep getting an error message when trying to download videos from this site....any help or insight would be greatly appreciated.

### Addon & CoApp version and details

{
  "version": "9.0.1.7a1",
  "target": "mozilla",
  "channel": "beta",
  "page_url": "https://membersarea.rawstrokes.com/knockout-and-marco-take-turns-fucking-danny-dinero.html",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [hls @ 0x13a804320] Unable to open key file https://hls2.videos.sproutvideo.com/4ace59a6dcbe8fa08bc1da346e02a9b6/c36aca4b14ade44375419b4fc7b7d531/video/1080.key\n[crypto @ 0x13ab21440] Unable to open resource: https://hls2.videos.sproutvideo.com/4ace59a6dcbe8fa08bc1da346e02a9b6/c36aca4b14ade44375419b4fc7b7d531/video/1080_00000.ts\n[crypto @ 0x13ab214c0] Unable to open resource: https://hls2.videos.sproutvideo.com/4ace59a6dcbe8fa08bc1da346e02a9b6/c36aca4b14ade44375419b4fc7b7d531/video/1080_00001.ts\n[crypto @ 0x13ab21440] Unable to open resource: https://hls2.videos.sproutvideo.com/4ace59a6dcbe8fa08bc1da346e02a9b6/c36aca4b14ade44375419b4fc7b7d531/video/1080_00002.ts\n[crypto @ 0x13ab21440] Unable to open resource: https://hls2.videos.sproutvideo.com/4ace59a6dcbe8fa08bc1da346e02a9b6/c36aca4b14ade44375419b4fc7b7d531/video/1080_00003.ts\n[crypto @ 0x13ab21440] Unable to open resource: https://hls2.videos.sproutvideo.com/4ace59a6dcbe8fa08bc1da346e02a9b6/c36aca4b14ade44375419b4fc7b7d531/video/1080_00004.ts\n[crypto @ 0x13ab21270] Unable to open resource: https://hls2.videos.sproutvideo.com/4ace59a6dcbe8fa08bc1da346e02a9b6/c36aca4b14ade44375419b4fc7b7d531/video/1080_00005.ts\n[hls @ 0x13a804320] Error when loading first segment 'https://hls2.videos.sproutvideo.com/4ace59a6dcbe8fa08bc1da346e02a9b6/c36aca4b14ade44375419b4fc7b7d531/video/1080_00000.ts'\n[in#0 @ 0x13a804220] Error opening input: Invalid data found when processing input\nError opening input file https://hls2.videos.sproutvideo.com/4ace59a6dcbe8fa08bc1da346e02a9b6/c36aca4b14ade44375419b4fc7b7d531/video/1080.m3u8?Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cHM6Ly9obHMyLnZpZGVvcy5zcHJvdXR2aWRlby5jb20vNGFjZTU5YTZkY2JlOGZhMDhiYzFkYTM0NmUwMmE5YjYvYzM2YWNhNGIxNGFkZTQ0Mzc1NDE5YjRmYzdiN2Q1MzEvKi5tM3U4P3Nlc3Npb25JRD02Mzg5M2RmZi1hYTc0LTRjOTItOTIwNC1jZDYxZDk1Njg1NjEiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjE3MTY4NTg4NTF9fX1dfQ__&Signature=N36wHtgQLw~FPymnRkK5lAAFf2~0dUaw~uw1o4bNXKpMtg9iBoxLmc9RnyQh3fvwYsx~4xi4ceEMyEy0wUzUEt83lAXMsy6RTM8R9FlbNR81T0ro0UZ3YD65eF~fJ2jFuAG847pVnfXlZHWfR74ILMCmXYoN2Vx6GR3-Oh1axZeoMTbbJ48pXckA3u3mEkt4qqG0~ub7ijhQPGUc3m-UnxZOR0XgWulo8nNXMt73OTQkmGb-eiSpnXADpJhBgImsVS~PtA8Rg1DLKIhPc6F5WQ4lT-t~Bl2BAIHWnklePac1a26aEF2H~kzD4StM-4ZK1IJ0AqN8Z2pmnGDxsDIMWA__&Key-Pair-Id=APKAIB5DGCGAQJ4GGIUQ&sessionID=63893dff-aa74-4c92-9204-cd61d9568561.\nError opening input files: Invalid data found when processing input\n",
    "id": "coapp_failure_3bb7995b-ab9a-4349-8498-7e3f5714f041",
    "downloadable_id": "downloadable_5942774958765993"
  }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I gather from the qualifier membersarea in that URL that you are talking about a paid subscription web site.  That kind of blocks anybody here from investigating this issue.

One of your error messages says there was an error opening a key file.  There have been a number of other reports in here, mostly answered by fellow user @mattmjs, that involved an error opening a key file.  Our best guess with this is that there is some sort of intentional error being generated by the web site & they don't want you to download whatever it is.  But that's a guess.  Without having access to the content, that's the best I can do.  No, I will not accept your login credentials even temporarily.  That is against VDH corporate policy & I take my cue from them.  Even as a non-employee, I think their policy is prudent.
